cardano-testnet/src/Testnet/Start/Cardano.hs

```diff
@@ -79,6 +90,19 @@ data ForkPoint
 startTimeOffsetSeconds :: DTC.NominalDiffTime
 startTimeOffsetSeconds = if OS.isWin32 then 90 else 15
 
+-- | Like 'cardanoTestnet', but using defaults for all configuration files.
+-- See 'cardanoTestnet' for additional documentation.
+cardanoTestnetDefault :: ()
+  => CardanoTestnetOptions
+  -> Conf
+  -> H.Integration TestnetRuntime
```

Comment:
Here, we're not mixing `CardanoTestnetOptions` and genesis files, which is good (as I explained elsewhere). This is nice for the vast majority of callers, that don't need to be exposes to a complex API.